core/icons/mutator_icon.ts

```diff
@@ -7,6 +7,7 @@
 // Former goog.module ID: Blockly.Mutator
 
 import type {BlockSvg} from '../block_svg.js';
+import {IBubble} from '../blockly.js';
```

Comment:
This can be import type, right? Here and elsewhere. Also, make sure you import it from the interface file and not from blockly.js

Reply:
Yep! Haven't fixed this yet, but also made notes to do so in my self-review. Will fix in a follow-up change.

Reply:
This has been addressed--marking as resolved to keep open only those threads that may need additional discussion.